core/profile.go

```diff
@@ -195,6 +195,12 @@ func (n *OpenBazaarNode) PatchProfile(patch map[string]interface{}) error {
 
 	repoProfile.NormalizeDataForAllSchemas()
 
+	if repoProfile.GetProtobuf().ModeratorInfo != nil && repoProfile.GetProtobuf().ModeratorInfo.Fee != nil {
+		if repoProfile.GetProtobuf().ModeratorInfo.Fee.FeeType == pb.Moderator_Fee_PERCENTAGE {
```

Comment:
Why not just make a single if condition rather than a nested conditional?

Reply:
Yeah I guess that could work too. I was just thinking about the nil checks as being separate.